cunumeric/module.py

```diff
@@ -413,7 +413,7 @@ def array(
     obj: Any,
     dtype: Optional[np.dtype[Any]] = None,
     copy: bool = True,
-    order: str = "K",
+    order: Union[OrderType, Literal["K"]] = "K",
```

Comment:
tried adding `"K"` to `OrderType` but that creates many cascading errors.

Reply:
This is the case in the original NumPy as well; some functions accept "A", "C" and "F", e.g. https://numpy.org/doc/stable/reference/generated/numpy.reshape.html, while others also accept "K", e.g. https://numpy.org/doc/stable/reference/generated/numpy.array.html.